src/features/common/Layout/MyForestContextV2.tsx

```diff
@@ -88,6 +102,14 @@ export const MyForestProviderV2: FC = ({ children }) => {
     slug: `${userInfo?.slug}`,
   });
 
+  useEffect(() => {
+    if (userInfo) {
+      setTreeTarget(userInfo.targets.treesDonated);
+      setRestoreTarget(userInfo.targets.areaRestored);
+      setConservTarget(userInfo.targets.areaConserved);
+    }
+  }, [userInfo]);
```

Comment:
I'm not clear why we need to maintain the targets as separate variables instead of simply using `userInfo`?

We've already stored part of the profile data in the `userInfo` and now we're segregating the data further.